node/grpc/utils.go

```diff
@@ -13,11 +13,16 @@ import (
 	"github.com/Layr-Labs/eigenda/node"
 	"github.com/consensys/gnark-crypto/ecc/bn254"
 	"github.com/consensys/gnark-crypto/ecc/bn254/fp"
+	"github.com/gammazero/workerpool"
 	"github.com/wealdtech/go-merkletree"
 	"github.com/wealdtech/go-merkletree/keccak256"
 	"google.golang.org/protobuf/proto"
 )
 
+var (
+	maxDeserializationWorkers = 32
```

Comment:
would be nice to expose this in operator config for tuning purposes

Reply:
sounds good, I didn't make it so as I was thinking the validator was not using flag yet and wanted to do it together separately later (validation is actually using flag)